src/hooks/useTransitionRouteParams.ts

```diff
@@ -0,0 +1,17 @@
+import {findFocusedRoute, useNavigationState} from '@react-navigation/native';
+import type {PublicScreensParamList, RootStackParamList} from '@libs/Navigation/types';
+import SCREENS from '@src/SCREENS';
+
+export default function useTransitionRouteParams() {
```

Comment:
It seems that `useTransitionRouteParams` will be used only to extract `exitTo`, so maybe we could create a `useExitTo` hook which will also simplify the code in the `HybridAppMiddlewareContext`?

Reply:
Yeah that's good idea, previously I was extracting other params so this is a legacy name now 😅 I'll adjust that!